.github/workflows/flakebox-flakehub-publish.yml

```diff
@@ -0,0 +1,35 @@
+# THIS FILE IS AUTOGENERATED FROM FLAKEBOX CONFIGURATION
```

Comment:
I see you copied it verbatim from some other repo. I guess that works. :D

Reply:
Yep, idk how to patch in new features after the fact with flakebox otherwise. I even made it read-only (if that worked) :laughing: